torchvision/prototype/datasets/_builtin/gtsrb.categories

```diff
@@ -0,0 +1,43 @@
+00000
```

Comment:
Are there no human-readable categories available? If not, you can replace this file (as well as the method that generates it), by passing `categories=[f"{label:05d}" for label in range(43)]` to the `DatasetInfo`.